config/config.yaml

```diff
@@ -35,6 +35,10 @@ container_registry:
 # careful: too many parameters might make your runs take a long time.
 
 algorithms:
+      - name: "local_neighborhood"
```

Comment:
Remember to remove `local_neighborhood` from the config file. You might also want to set the methods to be `include: true` as well.